service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -34,6 +35,7 @@ public static String buildSubjectLine(final String familyManCaseNumber,
             .collect(joining(", "));
     }
 
+    //Tech debt - Change this to use hearingBooking.toLabel()
```

Comment:
how about remove create a JIRA ticket in tech debt and remove the comment, otherwise is going to stay there forever

Reply:
made a JIRA https://tools.hmcts.net/jira/browse/FPLA-2616